ui/appui-react/src/test/frontstage/FrontstageManager.test.tsx

```diff
@@ -289,6 +289,35 @@ describe("FrontstageManager", () => {
     expect(newFrontstageDef).to.not.eql(frontstageDef);
   });
 
+  it("clearFrontstages should clear frontstages and fronstageProviders", async () => {
+    const frontstageProvider = new TestFrontstage();
+    InternalFrontstageManager.addFrontstageProvider(frontstageProvider);
+    expect(
+      InternalFrontstageManager.hasFrontstage(frontstageProvider.id)
+    ).toEqual(true);
+    const frontstageDef = await InternalFrontstageManager.getFrontstageDef(
+      frontstageProvider.id
+    );
+    expect(frontstageDef).toBeTruthy();
+    expect(InternalFrontstageManager.hasFrontstage(frontstageDef!.id)).toEqual(
+      true
+    );
+    await InternalFrontstageManager.setActiveFrontstage(frontstageDef!.id);
+    expect(InternalFrontstageManager.activeFrontstageId).toEqual(
+      frontstageDef!.id
+    );
+
+    InternalFrontstageManager.clearFrontstages();
+
+    expect(
+      InternalFrontstageManager.hasFrontstage(frontstageProvider.id)
+    ).toEqual(false);
+    expect(InternalFrontstageManager.hasFrontstage(frontstageDef!.id)).toEqual(
+      false
+    );
+    expect(InternalFrontstageManager.activeFrontstageId).toEqual("");
```

Comment:
There is a lot of overlap with other unit tests, let's test a single unit of code (described by the test name).
`defaultFrontstageConfig` in the below snipped can be imported from `"./FrontstageDef.test";`

```suggestion
    const frontstageProvider = new TestFrontstage();
    InternalFrontstageManager.addFrontstageProvider(frontstageProvider);
    InternalFrontstageManager.addFrontstage(defaultFrontstageConfig);
    expect(
      InternalFrontstageManager.hasFrontstage(frontstageProvider.id)
    ).toEqual(true);
    expect(
      InternalFrontstageManager.hasFrontstage(defaultFrontstageConfig.id)
    ).toEqual(true);

    InternalFrontstageManager.clearFrontstages();

    expect(
      InternalFrontstageManager.hasFrontstage(frontstageProvider.id)
    ).toEqual(false);
    expect(
      InternalFrontstageManager.hasFrontstage(defaultFrontstageConfig.id)
    ).toEqual(false);
```